cli/generic/forms/fields.py

```diff
@@ -57,6 +58,24 @@ def _deserialize(self, value, attr, obj, **kwargs):
         return value
 
 
+class StateIdentifierField(fields.Field):
```

Comment:
Is this a `header signature` or `block identifier` or `batch id`? Make it clear.

Reply:
No. This does not fit our code style.